src/main/java/edu/usf/cutr/grha/ProcessorMain.java

```diff
@@ -26,8 +27,10 @@
 public class ProcessorMain {
 
     public static void main(String[] args) {
-        if (args.length != 2) {
-            System.out.println("The first command-line parameter should be the GPSTest data filename with simple headers, and the 2nd parameter should be the filename of the Chicago dataset");
+        if (args.length != 3) {
```

Comment:
Please make the 3rd argument optional, and if it's not provided just output the GTFS to the current directory.